app/default/routes.py

```diff
@@ -21,6 +23,34 @@ def index():
     )
 
 
+@default_bp.route("/account/<account_id>")
+def dashboard(account_id):
+    response = requests.get(
+        f"{APPLICATION_STORE_API_HOST}/applications?account_id={account_id}"
+    ).json()
+    applications: list[ApplicationSummary] = [
+        ApplicationSummary.from_dict(application) for application in response
+    ]
+    return render_template(
+        "dashboard.html", account_id=account_id, applications=applications
+    )
+
+
+@default_bp.route("/account/<account_id>/new", methods=["POST"])
+def new(account_id):
+    new_application = requests.post(
+        url=f"{APPLICATION_STORE_API_HOST}/applications",
+        json={
+            "account_id": account_id,
+            "round_id": request.form["round_id"],
+            "fund_id": request.form["fund_id"],
+        },
+    )
+    return redirect(
+        url_for("tasklist", application_id=new_application.json().get("id"))
+    )
```

Comment:
Should there be some validation here (of the form/csrf etc) and of the response (which may be an error message without an id). Perhaps this logic should be a post on the dashboard route and any creation error message could be displayed there?

Reply:
As discussed offline, I believe the app is validating the csrf token in the HTML form. (you can validate this by removing it from the HTML form.) We could have more error handling but I think a 500 would be appropriate if something went wrong and currently we don't have a way to usefully log errors which is a broader issue.